test/e2e/pkg/testnet.go

```diff
@@ -108,6 +111,8 @@ type Testnet struct {
 	DefaultZone                                          string
 	PbtsEnableHeight                                     int64
 	PbtsUpdateHeight                                     int64
+	LanePriorities                                       []uint32
+	LanePrioritiesIndex                                  int
```

Comment:
What is this parameter for? I don't see this used anywhere

Reply:
Thanks, I'll remove it. It's dead code for a previous implementation.